### DISCUSSION TITLE
Dataset filesystem directory name

### DISCUSSION THREAD

**Discussion by onequery** [snuailab/waffle_hub#158] [Q&A] [answered]
The directory `datasets/` generated by `Dataset.load()`, `Dataset.from_coco()`, etc. are kind of uncomfortable to use when the users are managing their own dataset in their `datasets/` directory. It brings the directory name confliction.

 

Example:
```bash
# User's workspace
...
datasets/  # <- User cannot use this directory name for their own datasets 
models/
...
```

**Comment by ZeroAct** [comment, marked as answer]
You can change the directory name by setting `root_dir` arguement 👍.
